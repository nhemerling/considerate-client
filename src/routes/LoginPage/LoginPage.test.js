import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import LoginPage from './LoginPage';

describe('LoginPage component', () => {
  it('renders a LoginPage component', () => {
    const wrapper = shallow(<LoginPage />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
