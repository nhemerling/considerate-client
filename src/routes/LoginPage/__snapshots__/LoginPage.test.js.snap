// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPage component renders a LoginPage component 1`] = `
<section
  className="LoginPage"
>
  <h2>
    Login
  </h2>
  <LoginForm
    onLoginSuccess={[Function]}
  />
</section>
`;
